extensions/ui/org.eclipse.smarthome.ui.paper/pom.xml

```diff
@@ -29,8 +29,8 @@
         <version>1.0</version>
 
         <configuration>
-          <nodeVersion>v5.4.0</nodeVersion>
-          <npmVersion>3.3.12</npmVersion>
+          <nodeVersion>v8.1.2</nodeVersion>
+          <npmVersion>5.0.3</npmVersion>
```

Comment:
Just realized that these are build-time only and that there apparently no specific versions have to be given approval for.